ui/components/app/snaps/snap-delineator/snap-delineator.js

```diff
@@ -83,6 +88,8 @@ export const SnapDelineator = ({
             marginLeft={1}
             marginTop={0}
             marginBottom={0}
+            marginRight={1}
+            display="block"
```

Comment:
Why is this suddenly necessary?

Reply:
Also for the header fix, needed margin and the element has to be "block" for text overflow to work.